vignettes/in-packages.Rmd

```diff
@@ -182,17 +188,17 @@ my_function_inside_a_package <- function(...)
 
   if (tidyr_new_interface()) {
     # Freshly written code for v1.0.0
-    out <- tidyr::nest_legacy(df, x, y, z)
+    out <- tidyr::nest(df, data = one_of("x", "y", "z"))
   } else {
     # Existing code for v0.8.3
-    out <- tidyr::nest(df, c(x, y, z))
+    out <- tidyr::nest(df, x, y, z)
```

Comment:
I think this was just wrong?

Reply:
I think it worked but wasn't idiomatic previously

Reply:
Yeah ok. It is v0.8.3 code, written with a v1.0.0 sensibility.